report/src/presence_intro.tex

```diff
@@ -0,0 +1,23 @@
+\section{\label{sec:presence_intro}Room Occupancy Detection for Meeting Start and End Times}
+
+To address the problem defined in section \ref{TODO-SECTION 2.a.i AND 2.b.iii}, we propose an occupancy detection system (presence system) to identify when a meeting starts, defined as participants entering a room, and when it ends, defined as all participants leaving the meeting room.
+This section investigates various technologies for detecting room occupancy, evaluates their performance in the context of meeting start and end times, and determines the most suitable technology for our context of use.
+This technology will serve as the basis for the rest of the system, which will be described in the following sections.
+
+Several technologies can be utilized for detecting room occupancy, including infrared (IR) sensors\cite{woodward-2021}\cite{dodierBuildingOccupancyDetection2006}\cite{OccupancySensorMotion}, ultrasonic sensors\cite{woodward-2021}\cite{dodierBuildingOccupancyDetection2006}\cite{OccupancySensorMotion}, microwave sensors\cite{woodward-2021}, computer vision\cite{co2sensor}\cite{longoAccurateOccupancyEstimation2019}\cite{OccupancySensorMotion}, Bluetooth and Wi-Fi\cite{teissedre-2019}, pressure sensors\cite{OccupancySensorMotion}, acoustic sensors\cite{OccupancySensorMotion}, and CO2 sensors\cite{co2sensor}\cite{longoAccurateOccupancyEstimation2019}\cite{jinSensingProxyOccupancy2015}.\cite{faragherLocationFingerprintingBluetooth2015}\todo{Insert all the types you find and cite here. Remove TODO when this section is done.}
+Each technology has its strengths and weaknesses in terms of accuracy, cost, and performance.
+
+We have chosen accuracy, responsiveness, privacy, cost and environmental factors as evaluation criteria for occupancy detection technologies because they cover the key aspects necessary for successful implementation.
```

Comment:
This seems to come out of nowhere. We have not described our system in any detail, so we do not know key aspects of a succesful implementation. We have not presented any technologies so how and why do we pick any of these criteria for detection technologies?

Reply:
These criteria were created to serve as our evaluation criteria for judging the technologies, based on parameters we believe would be sufficient. The idea is to judge technologies based on these.

Reply:
Then describe them. We cannot just select a bunch of  criteria because they feel like they will be good.

Reply:
The description is on the line below.